src/components/FormInput.js

```diff
@@ -0,0 +1,10 @@
+const NewBook = () => (
+  <div className="newbook">
+    <form>
+      <input type="text" placeholder=" New Book" />
```

Comment:
- [ ] Great job so far. Remember, you will be working on creating a complete book with its author. Kindly ensure it Renders a form with inputs for the book title and the author of the book